src/components/drag-and-drop/with-drag/with-drag.js

```diff
@@ -57,11 +58,15 @@ const ItemSource = {
   },
 
   beginDrag(props, monitor, component) {
+    // Disable Text selection in Safari to show correct cursor
+    BrowserHelper.getDocument().onselectstart = function() { return false; };
```

Comment:
This seems like a safer solution. Inspiration found here - https://github.com/JedWatson/react-select/pull/1045/files/2745fb1303d93551a549e0c8ffa5076e9b230fad?short_path=04c6e90#diff-f4fddf467975bbb8c216d37f248bef72R125